perdido/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Perdido Demo</title>
+        <meta charset="utf-8" />
+        <script src="../jss.js"></script>
+        <script src="../jss-extend.js"></script>
+        <script src="../jss-nested.js"></script>
+        <script src="../jss-camel-case.js"></script>
+        <script src="../jss-default-unit.js"></script>
+        <script src="../jss-vendor-prefixer.js"></script>
+        <script src="../jss-props-sort.js"></script>
+        <script src="../perdido.js"></script>
+        <script src="./app.js"></script>
+    </head>
+    <body>
+        <header>
+            <h1><a href="https://gitub.com/wldcordeiro/perdido">Perdido Grid</a></h1>
+        </header>
+
+        <div class="description">
+            <a href="https://github.com/jsstyles/jss-examples/tree/gh-pages/perdido">View Source</a>
+            <p>Description here.</p>
+        </div>
+
+        <!-- Simple 1/4 -->
+        <section>
```

Comment:
I assume you are not using generated classes for simplicity reason. However I fear to make  a bad statement with global selectors.